### PR TITLE
feat(ui5-product-switch): Implement accessibility specification

### PR DESCRIPTION
**Pull request by dobrinyonkov**
FIXES: #2707

Additionally: 

UP and DOWN arrow keys navigation is updated, and limited to the product switch borders.

### CODE REVIEW

**Review by ilhan007 — DISMISSED** [review]
It looks good to me, but let's wait for @elenastoyanovaa to review from a11y perspective.
Just a reminder to update the a11y page afterwards:
https://wiki.wdf.sap.corp/wiki/pages/viewpage.action?pageId=2382053438

**Comment by dobrinyonkov** [comment]
I'm keeping this as a draft as it needs some further discussion in regards of the grid like navigation (arrows - up, down, left, right). The role menu -> menuitem and aria-setsize and posinset attributes may not be the most appropriate ones. I will discuss this with accessibility experts on the next round.

File: packages/fiori/src/ProductSwitch.js
Comment (nnaydenow) [inline packages/fiori/src/ProductSwitch.js:164]: if() {} else {} instead of if() {} if() {}

**Comment by nnaydenow** [comment]
Hi @ilhan007, 

For me the change looks okey. Can you approve it?

**Pull request merged**
